addons/mail/static/src/core/common/composer_model.js

```diff
@@ -75,6 +75,11 @@ export class Composer extends Record {
     /** @type {boolean} */
     forceCursorMove;
     isFocused = false;
+    autofocus = 0;
+
+    focus() {
+        this.autofocus++;
+    }
```

Comment:
Method `focus()` looks unnecessary indirection, makes code in component harder to understand, because we see `.focus()` and `.autofocus` and it's not immediately apparent that they are linked together in an immediate way.

Reply:
I feel it's convenient to call `composer.focus()` and have it focused instead of having an increment. How the composer is focused is a technical detail. `autofocus` should be private IMHO.

Reply:
This is more LOC and it adds a new method on a model, giving some incentive to add more behaviour on models, for which I'm cautiously against at the moment.

Reply:
I changed it and dislike it. Btw you only win 1 line of code since I can do `this.thread?.composer?.autofocus++;`.